crud/src/main/java/org/openmbee/sdvc/crud/controllers/projects/ProjectsController.java

```diff
@@ -111,7 +115,13 @@ public ProjectsResponse createOrUpdateProjects(
                 continue;
             }
 
+            if (!((schemas.getSchemas()).containsKey(json.getProjectType()))) {
```

Comment:
this is good, we also want it to be valid if getProjectType() is null, so either check it's not null here or set it to 'default' if it's null prior

Reply:
okay I added a check in the same line